Documented request tracing requirement for an active transaction on the scope to the .NET docs (#10118)
getsentry/sentry-docs: docs/platforms/dotnet/common/performance/instrumentation/automatic-instrumentation.mdx

- Populate the `sentry-trace` header on the request. This allows peer service to start a transaction by linking it to the current (assuming it's also instrumented with Sentry).
- Start a span named `GET https://example.com` which will track the corresponding HTTP operation on the current transaction.

To avoid depleting performance quota, Sentry will only create request spans when there is an active transaction on the scope. If there is no active transaction then you will need to create one before making an HTTP request, for request tracing to work:

```csharp

>   var transaction = SentrySdk.StartTransaction("tutorial", "example");
>   SentrySdk.ConfigureScope(scope => scope.Transaction = transaction);
>   var response = await httpClient.GetStringAsync("https://example.com");

Reviewer: ```suggestion
  var transaction = SentrySdk.StartTransaction("tutorial", "example");
  SentrySdk.ConfigureScope(scope => scope.Transaction = transaction);
  
  // The SentryHttpMessageHandler will automatically create a span for the request
  var response = await httpClient.GetStringAsync("https://example.com");
  
  transaction.Finish();
```
The snippets should be copy-pasta ready and fully functional.

- Populate the `sentry-trace` header on the request. This allows peer service to start a transaction by linking it to the current (assuming it's also instrumented with Sentry).
- Start a span named `GET https://example.com` which will track the corresponding HTTP operation on the current transaction.

> To avoid depleting performance quota, Sentry will only create request spans when there is an active transaction on the scope. If there is no active transaction then you will need to create one before making an HTTP request, for request tracing to work:
> 

Reviewer 2: ```suggestion
To avoid depleting performance quota, Sentry will only create request spans if there's an active transaction on the scope. If there's no active transaction, you'll need to create one before making an HTTP request for request tracing to work:

```
